tf_encrypted/keras/layers/convolutional.py

```diff
@@ -160,7 +144,9 @@ def build(self, input_shape):
     self.kernel = self.add_weight(kernel)
 
     if self.use_bias:
-      bias_shape = self.compute_output_shape(input_shape)[1:]
+      # Expand bias shape dimensions. Bias needs to have
+      # a rank of 3 to be added to the output
+      bias_shape = [self.filters, 1, 1]
```

Comment:
this seems weird -- it should be able to broadcast. okay with this code but it smells like a bug elsewhere